manifest.yaml

```diff
@@ -225,11 +228,12 @@ backup:
     mounts:
       BACKUP: /mnt/backup
       main: /root/.lightning
+    io-format: yaml
   restore:
     type: docker
     image: compat
     system: true
-    entrypoint: compat
+    entrypoint: /mnt/assets/backup-restore.sh
```

Comment:
I think you can remove the `args` key here, or leave it as `[]`, since those commands are included in `backup-restore.sh`

Reply:
Updated to `[]`